test/integration/dit_landing_page_test.rb

```diff
@@ -26,6 +26,5 @@ def and_i_can_see_the_translation_links
 
   def then_i_can_see_the_training_section
     assert page.has_selector?("h2", text: "Additional help and support")
-    assert page.has_selector?(".govuk-table__row", text: "Date")
```

Comment:
Why has this assertion been removed?

Reply:
Should there be a separate test to check that the table does display when "show=true"?